internal/parser/grammar.md

```diff
@@ -26,11 +26,14 @@ The parser directly implements the following grammar.
     AssignStatement                = "=" Expression .
     ReturnStatement                = "return" Expression .
     ExpressionStatement            = Expression .
-    Expression                     = LogicalExpression .
-    ExpressionSuffix               = { PostfixOperator } { PipeExpressionSuffix } { MultiplicativeExpressionSuffix } { AdditiveExpressionSuffix } { ComparisonExpressionSuffix } { LogicalExpressionSuffix } .
-    LogicalExpression              = UnaryLogicalExpression { LogicalExpressionSuffix } .
-    LogicalExpressionSuffix        = LogicalOperator UnaryLogicalExpression .
-    LogicalOperator                = "and" | "or" .
+    Expression                     = LogicalOrExpression .
+    ExpressionSuffix               = { PostfixOperator } { PipeExpressionSuffix } { MultiplicativeExpressionSuffix } { AdditiveExpressionSuffix } { ComparisonExpressionSuffix } { LogicalAndExpressionSuffix } { LogicalOrExpressionSuffix } .
+    LogicalOrExpression            = LogicalAndExpression { LogicalOrExpressionSuffix } .
```

Comment:
Can you group these together so that `LogicalOrExpression` is followed by `LogicalOrExpressionSuffix`?